### PR TITLE
mostly changes to exercises in lesson C

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

**Comment by ayan-usgs** [comment]
@aappling-usgs:

Could you give this PR a look?

File: vignettes/C_Clean.Rmd [hunk 1]
```diff
@@ -286,9 +286,9 @@ Let's practice some of these last functions with our Gages data.
 
 Next, we're going to pull in and merge additional water data from NWIS for NM GAGES-II sites. Keep track of your method by adding to your R script and commenting as you go. This exercise has several steps that each depend on the prior one- use your stickies and don't get frustrated! We'll work through these one at a time. If you complete a step and notice that your neighbor has not, see if you can answer any questions to help them get it done. 
 
-1)  Return to the code chunk in [Lesson B. Get](B_Get.html#reading-nwis-data-into-r) that accesses NWIS data. Modify that code chunk and add it to your R script to access phosphorus and also these additional physical and water quality variables at NM sites: temperature ('00010'), discharge ('00060'), and specific conductance ('00095'). Save these data in data frames named using the variable name and state abbreviation, e.g., 'discharge_fl'.
```
Comment (aappling-usgs) [inline vignettes/C_Clean.Rmd]: saving time is good, but there are implications in D. Explore ("create a pairs plot of the NM...", "Try a boxplot of discharge..."), E. Analyze ("let's take a look at the relationship between discharge, temperature, specific conductance, and phosphorus..."), and I don't know if anywhere else. Is it possible to leave these in now that you've made student efficiency gains through more guidance?

File: vignettes/C_Clean.Rmd [hunk 2]
```diff
@@ -311,9 +311,11 @@ disch_nm <- fix_wq_dates(disch_nm)
...
 
-5) Let's merge the water quality data we have just collected with the GAGES-II sites. Create a new data frame called 'gages_data_wq' by merging 'gages_data' with the NWIS variables we've already collected: phosphorus, temperature, discharge, and specific conductance. In the end, you should have a data frame that includes all rows and columns for GAGES-II sites for NM, in addition to five new columns: 'sample_dt', plus the four water quality variables we're merging. Hint: the number of rows in your expanded 'gages_data_wq' data frame will be much longer than in the original 'gages_data' data frame, and depending on the frequency of observations made for different variables, some variables may be more sparsely populated than others. 
+5) Merge the data from each of the four variable data frames so they can be easily viewed in a single data frame; this new data frame can be called `data_wq`. This action should preserve all rows from each of the original data frames.
```
Comment (aappling-usgs) [inline vignettes/C_Clean.Rmd]: nice addition

File: vignettes/C_Clean.Rmd (hunk 1 above)
Comment (aappling-usgs) [inline vignettes/C_Clean.Rmd]: oh - also, the exercise later still references four variables...?

Comment (ayan-usgs) [inline vignettes/C_Clean.Rmd]: > saving time is good

Thanks for catching that! You'll notice that later in Exercise 3, there are references to discharge and specific conductance. It was something I had removed, but forgot to add back in.

Comment (aappling-usgs) [inline vignettes/C_Clean.Rmd]: :+1:

**Comment by aappling-usgs** [comment]
looks good. i think we'll be well prepared to help students through this section now

**Pull request merged**